streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClient.java

```diff
@@ -217,7 +217,7 @@ private String ensureOneNodeIsReady(final List<Node> nodes) {
                 }
             }
             try {
-                kafkaClient.poll(streamsConfig.getLong(StreamsConfig.POLL_MS_CONFIG), Time.SYSTEM.milliseconds());
+                kafkaClient.poll(0, Time.SYSTEM.milliseconds());
```

Comment:
see https://github.com/apache/kafka/pull/3439#discussion_r125845931 and #3496

I guess it got merge a bit too fast.